src/features/surveys/components/SurveyEditor/blocks/OpenQuestionBlock.tsx

```diff
@@ -1,13 +1,235 @@
-import { Box } from '@mui/material';
-import { FC } from 'react';
+import AbcIcon from '@mui/icons-material/Abc';
+import { FormattedMessage as Msg } from 'react-intl';
+import SortIcon from '@mui/icons-material/Sort';
+import theme from 'theme';
+import { useIntl } from 'react-intl';
 import { ZetkinTextQuestion } from 'utils/types/zetkin';
+import {
+  BaseSyntheticEvent,
+  FC,
+  KeyboardEvent,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import {
+  Box,
+  ClickAwayListener,
+  MenuItem,
+  TextField,
+  Typography,
+} from '@mui/material';
+
+type TextQuestionPatchBody = Omit<ZetkinTextQuestion, 'required'>;
 
 interface OpenQuestionBlockProps {
-  question: ZetkinTextQuestion;
+  element: ZetkinTextQuestion;
+  inEditMode: boolean;
+  onEditModeEnter: () => void;
+  onEditModeExit: (question: TextQuestionPatchBody) => void;
+}
+
+enum FIELDTYPE {
+  MULTILINE = 'multiLine',
+  SINGLELINE = 'singleLine',
 }
 
-const OpenQuestionBlock: FC<OpenQuestionBlockProps> = ({ question }) => {
-  return <Box>{JSON.stringify(question)}</Box>;
+const OpenQuestionBlock: FC<OpenQuestionBlockProps> = ({
+  element,
+  inEditMode,
+  onEditModeEnter,
+  onEditModeExit,
+}) => {
+  const intl = useIntl();
+
+  const [typeField, setTypeField] = useState(
+    element.response_config.multiline === true
+      ? FIELDTYPE.MULTILINE
+      : FIELDTYPE.SINGLELINE
+  );
+
+  const [title, setTitle] = useState(element.question);
+  const [description, setDescription] = useState(element.description);
+
+  const [responseConfig, setResponseConfig] = useState(
+    element.response_config.multiline
+  );
+  const [focus, setFocus] = useState<
+    'title' | 'description' | 'responseConfig' | null
```

Comment:
'title' can actually be removed here since the focusing of the title field is handled by the callback ref!